api/src/main/java/org/openmrs/module/namephonetics/impl/NamePhoneticsServiceImpl.java

```diff
@@ -276,6 +277,23 @@ private boolean doPhoneticsContainPersonNameForField(Set<NamePhonetic> namePhone
 		}
 		return patients;
     }
+    
+    @Override
+   	public List<Person> findPerson(String givenNameSearch, String middleNameSearch, String familyNameSearch, String familyName2Search){
+           Set<Integer> idList = new HashSet<Integer>();
+           Set<PersonName> matches = getAllMatchingNamePhonetics(givenNameSearch, middleNameSearch, familyNameSearch, familyName2Search);
+           for (PersonName pn:matches){
+               idList.add(pn.getPerson().getPersonId());
+           }

```

Comment:
Looks good, though it would be nice to pull this out into a common util method that both findPerson and findPatient used.

Reply:
Would you like me to create a new issue based on this comment?